packages/storybook/src/utilities/storybook.ts

```diff
@@ -196,7 +197,7 @@ export const errorVisibleDescription = 'When set to `true`, the `error-text` mes
 export const dropdownPositionDescription = (options: {
     componentName: string
 }): string => `Controls the position of the dropdown relative to the ${options.componentName}.`;
-export const optionsDescription = `The \`${listOptionTag}\` items for the user to select from.`;
+export const optionsDescription = (options: { forSelect: boolean }): string => `The \`${listOptionTag}\` items for the user to select from.${options.forSelect ? ` Each ${listOptionTag} can also be grouped using the \`${listOptionGroupTag}\` element.` : ''} `;
```

Comment:
Based on the text, maybe the option name should be `includeGrouping` since it's about grouping docs and not about select docs